.github/scripts/load_skip_list.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -e
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+PROJECT_ROOT="$(cd "${SCRIPT_DIR}/../.." && pwd)"
+PYTHON_CMD="python3"
+
+SKIP_LIST_FILE="wild/tests/external_tests/mold_skip_tests.toml"
+CURRENT_SKIP_LIST="$(pwd)/current_skip_list.txt"
+
+${PYTHON_CMD} -c "
+import toml
+import sys
+try:
+    with open('${SKIP_LIST_FILE}', 'r') as f:
+        config = toml.load(f)
+    
+    all_skipped = []
+    for group_name, group_data in config.get('skipped_groups', {}).items():
+        all_skipped.extend(group_data.get('tests', []))
+    
+    for test in all_skipped:
+        test_name = test
+        if test_name.endswith('.sh'):
+            test_name = test_name[:-3]
+        # Convert special characters to match rstest behavior
+        converted_name = test_name.replace('+', '_').replace('.', '_').replace('-', '_')
+        print(converted_name)
+except Exception as e:
+    print(f'Error parsing TOML: {e}', file=sys.stderr)
+    sys.exit(1)
+" > "${CURRENT_SKIP_LIST}"
```

Comment:
Again, I think it'd be easier to work with as a proper Python script.

Reply:
I haven't looked through this change in any detail yet, but another possibility other than python would be Rust. It should be pretty quick to compile.

Reply:
Cargo script is an option but still requires nightly toolchain (or `RUST_BOOTSTRAP` hack): https://doc.rust-lang.org/nightly/cargo/reference/unstable.html#script